index.js

```diff
@@ -0,0 +1,214 @@
+    }
+    if (item.type === 'write') {
+      write(item);
+    }
+  } else if (item.code === 'Backspace' || item.code === 'Delete' || item.code === 'Tab' || item.code === 'Enter' || item.type === 'write') {
+    cursor = document.querySelector('textarea').selectionStart;
+    document.querySelector('textarea').value = document.querySelector('textarea').value.slice(0, document.querySelector('textarea').selectionStart) + document.querySelector('textarea').value.slice(document.querySelector('textarea').selectionEnd, document.querySelector('textarea').value.length);
+    document.querySelector('textarea').selectionStart = cursor;
+    document.querySelector('textarea').selectionEnd = cursor;
+    if (item.code !== 'Backspace' && item.code !== 'Delete') { enterSymbols(item); }
+  }
+}
+
+
+// PREPARATION
+
+function keyFill(language) {
+  localStorage.languageOfKeyboard = language;
+  document.querySelector('.keyboard').innerHTML = '';
+  objOfKeys.forEach((item) => {
+    const key = document.createElement('div');
+    key.className = `${item.size} ${item.code}`;
+    if (language === 'eng') {
+      if (item.view === 'double') {
+        key.innerHTML = `<p class="upLetter">${item.shiftEngValue}</p>`;
+        key.innerHTML += `<p class="centerLetter">${item.engValue}</p>`;
+      } else key.innerHTML = `<p class="centerLetter">${item.shiftEngValue}</p>`;
+    } else if (item.view === 'double') {
+      key.innerHTML = `<p class="upLetter">${item.shiftRuValue}</p>`;
+      key.innerHTML += `<p class="centerLetter">${item.ruValue}</p>`;
+    } else key.innerHTML = `<p class="centerLetter">${item.shiftRuValue}</p>`;
+    document.querySelector('.keyboard').append(key);
+  });
+}
+
+
+function createBase() {
+  const main = document.createElement('main');
+  document.querySelector('body').prepend(main);
+  const input = document.createElement('textarea');
+  document.querySelector('main').prepend(input);
+  const keyboard = document.createElement('section');
+  keyboard.className = 'keyboard';
+  document.querySelector('main').append(keyboard);
+}
+
+createBase();
+document.querySelector('textarea').focus();
+document.querySelector('textarea').setAttribute('placeholder', 'Keyboard designed for WINDOWS\nTo change the language click right shift and right alt!\nEnjoy ^__^');
+document.querySelector('body').setAttribute('onselectstart', 'return false');
+
+
+if (languageOfKeyboard === 'eng') {
+  keyFill('eng');
+} else {
+  keyFill('ru');
+}
+
+
+// START OF EVENTS
+
+document.querySelector('body').addEventListener('mousedown', (e) => {
+  cursor = document.querySelector('textarea').selectionStart;
+  const parent = e.target.parentElement;
+  const elem = e.target;
+  if (elem.tagName === 'DIV') elem.classList.add('buttonClick');
+  else if (parent.tagName === 'DIV') parent.classList.add('buttonClick');
+  objOfKeys.forEach((item) => {
+    if (elem.classList.contains(item.code) || parent.classList.contains(item.code)) {
```

Comment:
You have lots of checks for contsains. I think it would be better to create a function like this, it is just example:
![image](https://user-images.githubusercontent.com/31600165/79696301-937da380-8284-11ea-9b2a-fddb43797e31.png)

Here you can use `isEnyoneContains([elem, parent], [item.code])`